app/src/org/commcare/android/database/user/models/AUser.java

```diff
@@ -30,9 +31,13 @@
 import java.io.IOException;
 import java.util.Hashtable;
 
+/**
+ * Old Android user model now deprecated by combined User model in javarosa
+ */
```

Comment:
Would be good to detail a deprecation strategy. Do we keep this class around forever, or is there a timeline for removing it?